qa/TL1_tensorflow_dataset/test.sh

```diff
@@ -18,20 +18,29 @@ test_body() {
     is_compatible=$(python -c 'import nvidia.dali.plugin.tf as dali_tf; print(dali_tf.dataset_compatible_tensorflow())')
     if [ $is_compatible = 'True' ]; then
         # DALI TF DATASET run
-        nosetests --verbose -s test_dali_tf_dataset.py:_test_tf_dataset_other_gpu
-        nosetests --verbose -s test_dali_tf_dataset.py:_test_tf_dataset_multigpu
-        nosetests --verbose -s test_dali_tf_dataset_mnist.py
+        nosetests --verbose -s test_dali_tf_dataset_graph.py:_test_tf_dataset_other_gpu
```

Comment:
Out of curiosity, why do we need to list those tests explicitly?

Reply:
I made all tests that need more GPUs hidden (name starts with `_`), so they don't trigger in L0, where we run it on single GPU machine. Here I manually run those tests. 
Alternative is to create two files with tests, but the end result is the same.